tutorials/System.launchExternalProcess.md

```diff
@@ -1,13 +1,184 @@
 Runs an executable or batch file.
-If an unused uuid is provided in options, it will be used. If no uuid is provided, OpefinFin will assign a uuid.
```

Comment:
This seems worth leaving in somewhere, since it does allow flows such as:
- Pick a uuid
- Set up listener to fire when external process with given uuid is launched
- Launch with selected uuid
- When the external process is ready, do stuff.

This allows e.g. wrapping a non-openfin app with ExternalWindow, since the window is often not up and running and wrap-ready by the time the promise from `launchExternalProcess` resolves.

That said, typos are bad 😬

Reply:
Added UUID description back.